include/onnxruntime/core/framework/allocator_no_ort_symbol.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
```

Comment:
this file looks weird, it doesn't contain anything with Allocator, but named with allocator...